ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/PaymentStateMapper.java

```diff
@@ -12,12 +12,12 @@ public CCDPaymentState to(PaymentState paymentState) {
         return CCDPaymentState
             .builder()
             .status(paymentState.getStatus())
-            .finished(paymentState.isFinished())
+            .finished(paymentState.isFinished() ? "YES" : "NO")
             .build();
     }
 
     @Override
     public PaymentState from(CCDPaymentState payment) {
-        return new PaymentState(payment.getStatus(), payment.isFinished());
+        return new PaymentState(payment.getStatus(), payment.getFinished().equals("YES"));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move the "YES" string literal on the left side of this string comparison. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS1132)